src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TilesDocument.java

```diff
@@ -15,6 +15,7 @@
 import org.geoserver.ows.util.ResponseUtils;
 import org.geoserver.wms.WMS;
 import org.geowebcache.layer.TileLayer;
+import org.geowebcache.layer.meta.TileJSON;
```

Comment:
This feels like it should be split out - its vaguely related, but its really a different change.